src/rgw/rgw_sal_motr.cc

```diff
@@ -1844,12 +1844,15 @@ int MotrObject::write_mobj(const DoutPrefixProvider *dpp, bufferlist&& data, uin
 int MotrObject::read_mobj(const DoutPrefixProvider* dpp, int64_t off, int64_t end, RGWGetDataCB* cb)
 {
   int rc;
-  unsigned bs, actual, left;
+  unsigned bs, actual, left, start, bloff, block_start_off;
   struct m0_op *op;
   struct m0_bufvec buf;
   struct m0_bufvec attr;
   struct m0_indexvec ext;
 
+  start = off;
+  bloff = start;
```

Comment:
what is the initial value of start

Reply:
It will be same as off, if user is passing the byte range parameter then it would be starting offset otherwise it would be zero in normal get object scenario.

Reply:
`bloff = start;` is not required.

Reply:
Removed